src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/RawlsBillingProjectComponent.scala

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.dsde.rawls.dataaccess.slick
+
+import org.broadinstitute.dsde.rawls.model._
+
+trait RawlsBillingProjectComponent {
+  this: DriverComponent =>
+
+  import driver.api._
+
+  class RawlsBillingProjectTable(tag: Tag) extends Table[RawlsBillingProjectShort](tag, "BILLING_PROJECT") {
+    def projectName = column[String]("projectName", O.PrimaryKey)
```

Comment:
From what I've seen, strings are stored as text in the db by default. Since a lot of these things won't be arbitrarily long, do we want to be more efficient and store them as varchars?